models/classes/import/QtiTestImporter.php

```diff
@@ -42,9 +42,9 @@ class QtiTestImporter extends AbstractTestImporter
      * @param bool $itemMustBeOverwritten
      * @return \common_report_Report
      */
-    public function import(File $file, \core_kernel_classes_Class $class = null, $enableMetadataGuardians = true, $enableValidators = true, $itemMustExist = false, $itemMustBeOverwritten = false)
+    public function import(File $file, \core_kernel_classes_Class $class = null, $enableMetadataGuardians = true, $enableValidators = true, $itemMustExist = false, $itemMustBeOverwritten = false, $overwriteTest = false, ?string $itemClassUri = null)
     {
```

Comment:
```suggestion
    public function import(
        File $file,
        \core_kernel_classes_Class $class = null,
        $enableMetadataGuardians = true,
        $enableValidators = true,
        $itemMustExist = false,
        $itemMustBeOverwritten = false,
        $overwriteTest = false,
        ?string $itemClassUri = null
    ) {
```

Also, is it possible to add typehint for `$overwriteTest` ?

Reply:
fixed